Puara/ButtonAvnd.cpp

```diff
@@ -0,0 +1,44 @@
+#include "ButtonAvnd.hpp"
+
+namespace puara_gestures::objects
+{
+
+void ButtonAvnd::operator()()
+{
+  // 1. Getting the current button state from the input port
+  const int current_button_state = inputs.button_input.value;
+
+  // 2. Updating configurable parameters of the underlying puara_gestures::Button object
+
+  unsigned int desired_threshold = inputs.threshold_param.value;
+  if(this->impl.threshold != desired_threshold)
```

Comment:
just  

    this->impl.threshold = inputs.threshold_param;
    this->impl.countInterval = inputs.count_interval_param;
    this->impl.holdInterval = inputs.hold_interval_param;